### PR TITLE
NFC Non-Secure image support

### PR DESCRIPTION
**Pull request by kapi-no**
CHANGELOG.rst contains the list of changes.

Ref: NCSDK-4590

Signed-off-by: Kamil Piszczek <Kamil.Piszczek@nordicsemi.no>

### CODE REVIEW

**Comment by kapi-no** [comment]
@grochu, you suggested using release approach from the ```ble_controler``` library (revision based). Apart from the regular review, please check if this PR follows this new versioning scheme.

File: nfc/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,15 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nRF Connect SDK v1.3.0
```
Comment (grochu) [inline nfc/CHANGELOG.rst:8]: Should be "Master branch" now

File: nfc/CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,15 @@ Changelog
...
+**********************
+
+Added
+=====
+
+* Added support for Non-Secure firmware image to the NFC T2T and NFC T4T libraries.
+  Non-Secure image can only be run nRF devices that support Trusted Execution (e. g. nRF5340).
```
Comment (grochu) [inline nfc/CHANGELOG.rst:15]:
```suggestion
  Non-Secure image can only be run on nRF devices that support Trusted Execution (e. g. nRF5340).
```

File: nfc/CHANGELOG.rst (hunk 1 above)
Comment (kapi-no) [reply nfc/CHANGELOG.rst]: This is resolved in @b-gent patch

File: nfc/CHANGELOG.rst (hunk 2 above)
Comment (kapi-no) [reply nfc/CHANGELOG.rst]: Also resolved in @b-gent patch

**Comment by kapi-no** [comment]
Integrated changes from @b-gent and rebased this PR on top of master.

**Pull request merged**